### DISCUSSION TITLE
CTkTable in CTkScrollableFrame

### DISCUSSION THREAD

**Discussion by Pie49** [Akascape/CTkTable#114] [Q&A] [answered]
Seems that .place() method does not work, i don't have any problem with .pack().
Did i do something wrong ?
Thanks in advance

**Comment by Akascape** [comment, marked as answer]
@Pie49 `.place()` method is not usable in a scrollable frame, you can try with other widgets too.
I recommend using `.grid()` or `.pack()` method.

**Reply by Pie49** [reply]
Thank a lot for your quick response, got it.
